src/popup/router/components/Modals/BtnClose.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <button
```

Comment:
I think this button should have the same hover effect as the BtnIcon so maybe using BtnIcon here would be better to avoid unnecessary styling?

Reply:
close button have hover like is in the figma